pkg/continuoustest/config.go

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package continuoustest
+
+import (
+	"flag"
+
+	"github.com/grafana/dskit/log"
+)
+
+type Config struct {
+	ServerMetricsPort   int                       `yaml:"server_metrics_port"`
+	LogLevel            log.Level                 `yaml:"log_level"`
```

Comment:
These 2 settings should be removed, because we should use the Mimir ones.

Reply:
Leaving in for the backwards compatibility of `cmd/mimir-continuous-test/main.go` for right now and will eventually remove. Unused in module now.